recipes-bsp/firmware-imx/firmware-nxp-wifi_1.1.bb

```diff
@@ -168,5 +173,3 @@ RDEPENDS:${PN}-nxpiw612-sdio += "${PN}-nxp-common"
 RPROVIDES:${PN}-nxpiw612-sdio = "linux-firmware-nxpiw612-sdio"
 RREPLACES:${PN}-nxpiw612-sdio = "linux-firmware-nxpiw612-sdio"
 RCONFLICTS:${PN}-nxpiw612-sdio = "linux-firmware-nxpiw612-sdio"
-
-COMPATIBLE_MACHINE = "(imx-generic-bsp)"
```

Comment:
I think this should be removed since this recipe should not be restricted to NXP machines, as explained on the commit message.